tests/test_helpers.py

```diff
@@ -17,6 +17,10 @@
       "--date", "2020-04-29", "-c", "afriend", "-d", "more work",
       "-m", "ascopatz", "-g", "SymPy-1.1"],
      "20ly_newprojectum has been added in projecta\n"),
+    (["helper", "a_proposal", "a new proposal","100.0","To destroy numbers",
```

Comment:
maybe load two things for things that take lists for slightly better coverage?

Reply:
done!